kafka-connect/src/main/scala/net/manub/embeddedkafka/connect/EmbeddedKafkaConnect.scala

```diff
@@ -0,0 +1,106 @@
+package net.manub.embeddedkafka.connect
+
+import java.io.File
+
+import org.apache.kafka.common.utils.Time
+import org.apache.kafka.connect.connector.policy.ConnectorClientConfigOverridePolicy
+import org.apache.kafka.connect.runtime.isolation.Plugins
+import org.apache.kafka.connect.runtime.rest.RestServer
+import org.apache.kafka.connect.runtime.standalone.{
+  StandaloneConfig,
+  StandaloneHerder
+}
+import org.apache.kafka.connect.runtime.{Connect, Worker, WorkerConfig}
+import org.apache.kafka.connect.storage.MemoryOffsetBackingStore
+import org.apache.kafka.connect.util.ConnectUtils
+
+import net.manub.embeddedkafka.{
+  EmbeddedKafka,
+  EmbeddedKafkaConfig,
+  EmbeddedKafkaSupport
+}
+
+import scala.jdk.CollectionConverters._
+
+/**
+  * Helper trait for running Kafka Connect server.
+  * Use `.startConnect` to start the server.
+  */
+trait EmbeddedKafkaConnect
+    extends EmbeddedKafkaConnectSupport[EmbeddedKafkaConfig]
```

Comment:
Shouldn't this type param be `EmbeddedConnectConfig` instead?

Reply:
I'd say it's fine. The same way as it is in the EmbeddedKafkaStreams trait

Reply:
Yes, but Streams requires no additional config property, while Connect does. Schema Registry, for one, references its own config (see [here](https://github.com/embeddedkafka/embedded-kafka-schema-registry/blob/master/embedded-kafka-schema-registry/src/main/scala/net.manub.embeddedkafka/schemaregistry/EmbeddedKafka.scala#L13)).

Reply:
You mean adding the case class extending from ``EmbeddedKafkaConfig`` and extending its corresponding trait, right?

Reply:
I'm sorry, I've just noticed that `EmbeddedConnectConfig` does not extend `EmbeddedKafkaConfig` (as Schema Registry config does), but it's a wrapper instead.